docs/headless-cms/08-01-add-clusterdocstopic.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Add a ClusterDocsTopic
+type: Tutorial
+---
+
+Headless CMS allows you to extend documentation topics in the Console UI using ClusterDocsTopic custom resources.
+
+When you prepare ClusterDocsTopics, make sure that:
+- ClusterDocsTopics have a proper [custom resource definition](#custom-resource-clusterdocstopic).
+- Documents start with metadata, including obligatory `Title` for the document name and optional `Type` for any additional document grouping.
+- Documents are written in Markdown and follow a simple Markdown syntax. Any customizations can result in incorrect images or links rendering. To avoid such issues, use our [content guidelines](https://github.com/kyma-project/community/tree/master/guidelines/content-guidelines).
+
+This tutorial shows how you can customize the `docs-ui` view that is available in the Console UI under the ![](./assets/docs.png) icon on the top navigation panel. The purpose of this tutorial is to create a new **Prometheus** documentation section that contains **Concepts** and **Guides** topic groups and a set of Markdown subdocuments. The document sources used in the ClusterDocsTopics point to specific topics in the official Prometheus documentation.  
+
+>**NOTE:** When you apply multiple ClusterDocsTopics, sort them using the **order** label. Otherwise, the Console UI displays the topics in the order in which you apply them.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Steps
+
+1. Install Kyma.
+
+2. In the terminal, apply these ClusterDocsTopic custom resources:
+
+```yaml
```

Comment:
please add here comments to the CRs, what specific parts mean, especially the labels are important:
- what cms.kyma-project.io/group-name will do 
- what cms.kyma-project.io/order: "1" will do
- what is displayName: "Concepts" all about
- and general comment to sources, explains that by indicating `markdown` type the fetched source content will be analyzed by special Markdown service that will extract front matter metadata for the files and later they are added to the status. Also, emphasize again that the source is pointing to official Prometheus docs